### PR TITLE
Create zh_cn for  mod and Patchouli book.

### PR DESCRIPTION
**Pull request by IwasConfused**

### CODE REVIEW

**Comment by fzzyhmstrs** [comment]
Thank you very much for all the work you've put into this translation! :).

@chronosacaria , this is an updated PR of the one I mentioned in discord a couple days ago. If you have some spare time I would really appreciate a review of the translations.

**Comment by chronosacaria** [comment]
Thanks for the ping reminder, because I would have, honestly, forgotten. Lemme finish some paperwork and I'll take a look at this right afterwards

**Comment by chronosacaria** [comment]
So, there is a subtle change that I would suggest, which, whilst the create a more literal translation, I would argue that these changes are more accurate to the letter of the source whilst still retaining the spirit of the source.

For the `zh_cn.json` file, I suggest the following change:
 - Line 65: 璀璨之约 --> 许诺宝石

Otherwise, this is an awesome translation and a massive undertaking which is greatly appreciated :D

**Comment by IwasConfused** [comment]
@chronosacaria Thank you very much for your suggestions! 

I was not sure of the exact meaning of "Promising" here. The original intention was to make the translation of this advancement and the next advancement correspond, so it may not be consistent with the original. 

I will modify the translation of both advancements according to the suggestion. I think it's better to ask the exact meaning of this: 
the "Promising" here refers to the gem itself or the behavior of the player acquiring the gem? The same question applies to "As Promised". 

Once I get an exact answer, I will try to give more appropriate translation.

**Comment by chronosacaria** [comment]
That's actually a really good question that I had not really considered 🤔 I thought that I knew the answer to that question, which is why I offered the suggestion that I did, but you're point makes me realise that I am actually unsure 😅

**Comment by fzzyhmstrs** [comment]
I'll try to explain my usage of the phrases in question:

Both advancements in question are based on the "gem of promise" item. This item uses promise to mean "this gem can be better than it is; it promises more". "Gem of opportunity" might be an alternate phrasing.

The first advancement "Promising" is a play on the gems name. It uses "promising" to mean "this item looks good", "this item looks promising".

The second advancement "as promised" is more directly related to the name of the item. The gem promises you more, and it has delivered on its promise.

**Comment by IwasConfused** [comment]
> I'll try to explain my usage of the phrases in question:
> 
> Both advancements in question are based on the "gem of promise" item. This item uses promise to mean "this gem can be better than it is; it promises more". "Gem of opportunity" might be an alternate phrasing.
> 
> The first advancement "Promising" is a play on the gems name. It uses "promising" to mean "this item looks good", "this item looks promising".
> 
> The second advancement "as promised" is more directly related to the name of the item. The gem promises you more, and it has delivered on its promise.

Thanks for explaining! I will change the translation according to them! It may be difficult to convey the polysemy of "promise", but I will try.

**Comment by IwasConfused** [comment]
> I'll try to explain my usage of the phrases in question:
> 
> Both advancements in question are based on the "gem of promise" item. This item uses promise to mean "this gem can be better than it is; it promises more". "Gem of opportunity" might be an alternate phrasing.
> 
> The first advancement "Promising" is a play on the gems name. It uses "promising" to mean "this item looks good", "this item looks promising".
> 
> The second advancement "as promised" is more directly related to the name of the item. The gem promises you more, and it has delivered on its promise.

Thanks for explaining! I will change the translation according to them! It may be difficult to convey the polysemy of "promise", but I will try.

**Comment by Cyenoch** [comment]
Missing brackets at
-----------------------------------------------------------------------**HERE**
"...拿着$(l:amethyst_imbuement:imbuing/gem_of_promise机遇宝石$(/l)$()。..."

https://github.com/IwasConfused/ai/blob/377a979b956eb9a8ccc954b3c96f26975bb7cb20/src/main/resources/data/amethyst_imbuement/patchouli_books/glistering_tome/zh_cn/entries/scepter_upgrading/aspect_gems.json#L6-L9

File: src/main/resources/data/amethyst_imbuement/patchouli_books/glistering_tome/zh_cn/entries/scepter_upgrading/aspect_gems.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{    
+"name": "特化宝石",
+"icon": "amethyst_imbuement:blazing_gem",    
+"category": "amethyst_imbuement:scepter_upgrading",    
+"pages": [        
+{            
+"type": "patchouli:text",            
+"text": "特化宝石是你在升级之道上制作各种法杖的重要道具。它们是通过以各种方式与环境交互而产生的。$(br2)$(bold)任何特化宝石的制造都需要副手拿着$(l:amethyst_imbuement:imbuing/gem_of_promise机遇宝石$(/l)$()。"
```
Comment (fzzyhmstrs) [inline src/main/resources/data/amethyst_imbuement/patchouli_books/glistering_tome/zh_cn/entries/scepter_upgrading/aspect_gems.json:8]: See comment made by Cyenoch

**Review by fzzyhmstrs — APPROVED** [review]
With the fixed parentheses, this translation looks good! Approving based on fix of brace and results of Chronos's review.

**Pull request merged**